background_removal.py

```diff
@@ -37,7 +37,7 @@ def download_model():
 def ml_predict(image):
     with graph.as_default():
         # Add a dimension for the batch
-        prediction = model.predict(image[None, :, :, :])
+        prediction = model.predict(image)
```

Comment:
ideally , `ml_predict` should be pure enough and not burdened with adding batch. It should be handled by calling function `predict1`